src/components/Home/Hero.js

```diff
@@ -49,7 +49,7 @@ export default function Hero() {
                         <span className="inline-block">Product OS</span>
                     </h1>
                     <h2 className={heading('subtitle', 'primary', 'my-6 !text-black/50')}>
-                        A suite of product and data tools. Built on the modern data stack.
+                       Everything engineers need to build better products
```

Comment:
```suggestion
                       We help engineers build better products
```

Reasoning:

* we cannot list all our stuff here - the bit below does a great job of this
* anything tool-based therefore is vague - we've tried lots of ideas. we aren't "everything you need to build better products", we're like 50% of it, so this is misleading/feels bad at brand
* I really like how this suggestion elevates us - we're more mission driven / institutional than our lame competitors, and it does it in a simple way that perfectly fits our strategy

Reply:
Agree with your points, but in a vacuum I would read 'We help engineers build better products' and think 'Oh, it's some sort of analytics agency'

Reply:
Personal opinion is that it doesn't actually explain _what_ we do - it leans more on the side of vague marketing-y speak. (I'm not disagreeing the current one can't be improved.)

It's also just slightly wordier than Maxpanel's homepage headline, making this the GPT-style version of that phrase (ie: more verbose).

I think this line will be perfect for 2+ years from now, when everyone knows about PostHog.

A few years into Slack being the hot new thing, their headline was, "So, we tried Slack..." (inferring it was a life-changing experience). But this wasn't their first headline - they needed to be well-enough known that the headline was backing up the image people already had about the company.

This feels like a tagline for that stage.

(Yet despite my wordiness, I don't feel strongly enough to close this. 😉)

Reply:
Basically agree with the above comments. None of the options we have so far really work for me. 

I'd add that we really need to look at both the header and sub-header together, as neither covey either benefits or mission / values.

A quick random selection for comparison:

![Screenshot 2023-03-13 at 11 29 44](https://user-images.githubusercontent.com/92976667/224690452-41d0b492-1ff6-4375-9056-5918f417f442.png)

![Screenshot 2023-03-13 at 11 28 18](https://user-images.githubusercontent.com/92976667/224690479-31c71f80-d7a6-43c3-ba55-59ab2f8f58fa.png)

![Screenshot 2023-03-13 at 11 28 24](https://user-images.githubusercontent.com/92976667/224690491-ec40354a-37ec-417a-9c4b-fda5dcbc2622.png)

![Screenshot 2023-03-13 at 11 28 35](https://user-images.githubusercontent.com/92976667/224690595-9ad2f4d7-6851-47df-8b0d-cda1612e0520.png)

![Screenshot 2023-03-13 at 11 28 45](https://user-images.githubusercontent.com/92976667/224690630-04795dd0-293a-459b-a45c-37c232ccf6fa.png)

How does our head+sub-head compare to these ones? What do we like / dislike about them?

Reply:
@jamesefhawkins @corywatilo please could you make a final decision in your 1-1 this week? I think you have enough input now to make the final call!

Reply:
Sidenote: I haven't really had time to think very heavily about this. I will have a long think before forming a more useful opinion, if it's needed.